jobbergate-api/jobbergate_api/apps/job_scripts/routers.py

```diff
@@ -191,8 +175,19 @@ async def job_script_create(
     except INTEGRITY_CHECK_EXCEPTIONS as e:
         raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_ENTITY, detail=str(e))
 
+    else:
+
+        s3man_jobscripts.put(
```

Comment:
I have a couple of issues with this:
First, there's no exception handling around the `put()` method here. What happens if the user doesn't have write permission on the bucket or any other s3 access error? I think we will need a column to indicate if the files were successfully uploaded to s3 or not. Otherwise, we could easily have what appears to be a valid job_script but that has no script uploaded.

Reply:
Yeah, I'll implement a try/except and throw an HTTP exception if `s3man_jobscripts.put` goes wrong.

I think that a status column is not necessary if we implement a roll-back in the database when the endpoints create and put face errors with S3, as discussed in other points in this review.

Reply:
This operation is now done in a try-except block, providing a message if anything goes wrong. The exception handling for no such bucket was removed.